cms/envs/common.py

```diff
@@ -340,7 +340,6 @@
     'ENABLE_ENROLLMENT_RESET': False,
     'DISABLE_MOBILE_COURSE_AVAILABLE': False,
 
-    # .. toggle_name: FEATURES['ENABLE_CHANGE_USER_PASSWORD_ADMIN']
```

Comment:
@regisb: Should this have caused a failure?  Does that mean the new rules are not yet being used?  I didn't look into it, just noting this.

Reply:
@robrap The new linting rule did cause a failure: https://build.testeng.edx.org/job/edx-platform-quality-pipeline-pr/23211/pylint/new/folder.-1597087698/ However I have no idea why this did not cause the build to fail :-/

Reply:
@regisb: I think all the violations are being associated with line 1, so diff-quality isn't associating it to the change I made. Is there any way to have the violations more closely associated to the line that is failing?  Eventually we will no longer use diff-quality, and this may be less of an issue, but it would still be nice if the violations had accurate line numbers.

Reply:
Good catch! I didn't realize the new errors were associated to line 1. This might be due to the fact that the new checkers run `visit_module()`. I'll investigate.